src/dune_rules/pkg_rules.ml

```diff
@@ -277,7 +284,7 @@ module Pkg = struct
         ]
   ;;
 
-  let deps_closure t = top_closure t.depends
+  let deps_closure t = top_closure (t.depends @ t.inherited_post_depends)
```

Comment:
Aren't `t.inherited_post_depends` added twice now? First here and then in `top_closure`?